src/components/plotly/bar-line/index.js

```diff
@@ -100,7 +108,8 @@ BarLine.propTypes = {
   x: PropTypes.string.isRequired,
   y: PropTypes.arrayOf(PropTypes.string).isRequired,
   showTicks: PropTypes.bool,
-  showAxisTitles: PropTypes.bool,
+  showAxisTitles: PropTypes.objectOf(PropTypes.bool),
+  axisTitles: PropTypes.objectOf(PropTypes.string),
```

Comment:
maybe for next iteration we can use shape (exact) to make this a bit more rigid and self-documenting (VS Code does a good job inferring prop types for component usage doc):

```suggestion
  showAxisTitles: PropTypes.shape({
    x: PropTypes.bool,
    y: PropTypes.bool,
    y2: PropTypes.bool,
  }),
  axisTitles: PropTypes.shape({
    x: PropTypes.string,
    y: PropTypes.string,
    y2: PropTypes.string,
  }),
```

Reply:
@woozyking do you mind if I change it now here?

Reply:
absolutely!